clu/src/lib.rs

```diff
@@ -295,6 +295,7 @@ mod tests {
     }
 
     #[test]
+    #[ignore]
     fn test_generate_wg_key() {
```

Comment:
Its not pretty but IMO `create_wg_keypair` doesn't go through the usual "mock" interface of KernelInterface like other methods. It just calls `Command::new`. It would make more sense to use the other interface to verify that `wg genkey` and `wg pubkey` is called.

Reply:
well I'm the guilty party that wrote that test, at the time I was interested to see if my stdin/out shenanigans worked for real and not just with the mocker.

Reply:
Saved for posterity: https://github.com/althea-mesh/althea_rs/issues/320